doc/website/blog/tbd-announcing-versioned-migrations.md

```diff
@@ -0,0 +1,346 @@
+---
+title: Announcing Versioned Migrations
+author: MasseElch
+authorURL: "https://github.com/masseelch"
+authorImageURL: "https://avatars.githubusercontent.com/u/12862103?v=4"
+image: tbd
+---
+
+When [Ariel](https://github.com/a8m) released Ent v0.10.0 at the end of January,
+he [introduced](2022-01-20-announcing-new-migration-engine.md) a new migration engine for Ent based on another
+open-source tool called [Atlas](https://github.com/ariga/atlas). A month ago we added support for Atlas to generate
```

Comment:
In this post, I want to showcase a new kind of migration workflow that has recently been added to Atlas and Ent. We call it "versioned migration authoring" and it's an attempt to combine the simplicity and expressiveness of the declarative approach with the safety and explicitness of versioned migrations. With versioned migration authoring, users still declare their desired state and use the Atlas engine to plan a safe migration from the existing to the new state. However, instead of coupling the planning and execution of this plan, it is instead written into a file which can be checked into source control, fine tuned manually and reviewed in normal code review processes.